data/ecosystem/adopters.yaml

```diff
@@ -67,9 +66,10 @@
   contact: ''
 - name: Heroku
   url: https://heroku.com
-  components: [Ruby]
-  reference: ''
-  contact: ''
+  components: [Collector, Go, Ruby, JavaScript, Operator]
+  reference: 'https://www.heroku.com/blog/opentelemetry-basics-on-heroku-fir/'
+  referenceTitle: blog post
+  contact: 'https://www.heroku.com/contact/'
```

Comment:
Reading this, I would argue this is an integration and not adoption:

Adoption would mean "here is how heroku uses OpenTelemetry internally to ensure that services are up and running"

Integration means "here is how heroku provides OpenTelemetry to end users"

Reply:
@svrnm That makes total sense. I've update the PR to add Heroku to the integration registry. I also tweaked the adopters to link off to the talk I presented at KubeCon NA24 on that topic.

Please let me know if there are any other tweaks you would like to see.